src/main/java/org/gwtproject/dom/client/MediaElement.java

```diff
@@ -54,47 +54,56 @@
   /**
    * Constant returned from {@link #getReadyState()}.
    */
-  public static int HAVE_NOTHING;
+  @JsOverlay
+  public static int HAVE_NOTHING = 0;
```

Comment:
The original intent I believe was that this MediaElement class was actually a JsType for HTMLMediaElement, which actually has these fields - try `HTMLMediaElement.HAVE_METADATA` in the browser console.

Since it isn't, we do need this overlay, or we need @JsProperty to point at the proper class in JS. 

If we make them overlays, maybe make the fields final too?